scripts/makebumpver

```diff
@@ -231,6 +228,10 @@ class MakeBumpVer:
 
     def _queryBug(self, bugid):
         if not self.bz:
+            sys.stdout.write("Importing bugzilla module...")
```

Comment:
I don't think we want it logged.

Reply:
Fair enough. Maybe we could drop more of the logging too...